build.sbt

```diff
@@ -40,7 +40,7 @@ lazy val article_api = (project in file(".")).
       "javax.servlet" % "javax.servlet-api" % "3.1.0" % "container;provided;test",
       "org.scalatra" %% "scalatra-json" % Scalatraversion,
       "org.scalatra" %% "scalatra-scalatest" % Scalatraversion % "test",
-      "org.json4s"   %% "json4s-native" % "3.3.0",
+      "org.json4s"   %% "json4s-native" % "3.5.0",
```

Comment:
Ser ikke ut som veldig store endringer: https://github.com/json4s/json4s/compare/v3.5.0...3.4

Forøvrig, så funker scalatra 2.5 særs dårlig med eldre versjoner av json4s. Derav denne oppgraderingen også.